docs/release_notes/v0.18.4-beta.md

```diff
@@ -0,0 +1,52 @@
+# Spice v0.19.0-beta (Oct 7, 2024)
+
+Spice v0.19.0-beta brings performance improvements for accelerators and expanded TPC-DS coverage. A new Azure Blob Storage data connector has also been added.
+
+## Highlights in v0.19.0-beta
+
+**Improved TPC-DS Coverage**: Enhanced support for TPC-DS derived queries.
+
+**CLI SQL REPL**: The CLI SQL REPL (`spice sql`) now supports multi-line editing and tab indentation. Note, a semi-colon ';' is now required for each executed SQL block.
+
+**Azure Storage Data Connector**: A new Azure Blob Storage data connector (`abfs://`) has been added, enabling federated SQL queries on files stored in Azure Blob-compatible endpoints, including Azure BlobFS (`abfss://`) and Azure Data Lake (`adl://`). Supported file formats can be specified using the `file_format` parameter.
+
+Example `spicepod.yml`:
+
+```yaml
+datasets:
+  - from: abfs://foocontainer/taxi_sample.csv
+    name: azure_test
+    params:
+      azure_account: spiceadls
+      azure_access_key: abc123==
+      file_format: csv
+```
+
+For a full list of supported files, see the [Object Store File Formats](/components/data-connectors/index.md#object-store-file-formats) documentation.
+
+For more details, see the [Azure Blob Storage Data Connector documentation](https://docs.spiceai.org/components/data-connectors/github).
+
+## Breaking Changes
+
+- **GitHub Data Connector:** Pull Requests column `login` has been renamed to `author`.
+- **GitHub Data Connector:** Pull Requests and Issues column `labels` has been changed to a plain string array.
```

Comment:
@peasee  I was under impression that labels were already a Utf8 Array
https://docs.spiceai.org/components/data-connectors/github#querying-github-pull-requests
```
sql> select title, state, labels from spiceai.issues where title like '%duckdb%'
+-----------------------------------------------------------------------------------------------------------+--------+----------------------+
| title                                                                                                     | state  | labels               |
+-----------------------------------------------------------------------------------------------------------+--------+----------------------+
| Limitation documentation duckdb accelerator about nested struct and decimal256                            | CLOSED | [kind/documentation] |
| Inconsistent duckdb connector params: `params.open` and `params.duckdb_file`                              | CLOSED | [kind/bug]           |
| federation across multiple duckdb acceleration tables.                                                    | CLOSED | []                   |
| Integration tests to cover "On Conflict" behaviors for duckdb accelerator                                 | CLOSED | [kind/task]          |
| Permission denied issue while using duckdb data connector with spice using HELM for Kubernetes deployment | CLOSED | [kind/bug]           |
+-----------------------------------------------------------------------------------------------------------+--------+----------------------+

Time: 0.011877542 seconds. 5 rows.
```

Reply:
https://github.com/spiceai/spiceai/pull/2452/files